docs/user/bigtable/usage.rst

```diff
@@ -10,13 +13,28 @@ are obtained from this file. For more information, see `Connecting to Cloud Bigt
 .. code-block:: java
 
     Map<String, Serializable> parameters = new HashMap<>;
-    parameters.put("bigtable.table.name", "geomesa");
+    parameters.put("bigtable.catalog", "geomesa");
     org.geotools.data.DataStore dataStore =
         org.geotools.data.DataStoreFinder.getDataStore(parameters);
 
-The data store takes one parameter:
+.. _bigtable_parameters:
+
+Parameters
+----------
+
+The data store takes several parameters (required parameters are marked with ``*``):
 
-* ``bigtable.table.name`` - the name of the Bigtable table that stores feature type data
+==================================== ======= ====================================================================================
+Parameter                            Type    Description
+==================================== ======= ====================================================================================
+``bigtable.catalog *``               String  The name of the GeoMesa catalog table (previously ``bigtable.table.name``)
```

Comment:
Do we have a lookup table between old and new?

Reply:
not in the docs, if that's what you mean, other than these tables. in the code, the parameter instances keep track of any deprecated keys, and will log a warning if you use one

Reply:
I'd say that a short migration guide might be justified.  If we ever remove the deprecated keys, it'll be a breaking change.

Reply:
i figured we could do something in the release notes and point people to these tables...

Reply:
Maybe we should start a version controlled 'migration' guide?  We can pre-pend instructions for going from version {n-1} to {n}.

Specifically, I'm thinking of: http://docs.geotools.org/stable/userguide.old/welcome/upgrade.html.